src/app/zap_cluster_list.json

```diff
@@ -324,7 +324,7 @@
         "LAUNDRY_WASHER_CONTROLS_CLUSTER": ["laundry-washer-controls-server"],
         "LAUNDRY_DRYER_CONTROLS_CLUSTER": ["laundry-dryer-controls-server"],
         "WEB_RTC_TRANSPORT_PROVIDER_CLUSTER": [],
-        "WEB_RTC_TRANSPORT_REQUESTOR_CLUSTER": [],
+        "WEB_RTC_TRANSPORT_REQUESTOR_CLUSTER": ["webrtc-requestor-server"],
```

Comment:
Please rename the folder name to 'webrtc-transport-requestor-server' to match the cluster name